.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+version: 2.1
+
+orbs:
+  security: studion/security@2.0.0
+
+pr-filters: &pr-filters
+  branches:
+    ignore: master
+
+trunk-filters: &trunk-filters
+  branches:
+    only: master
+    
+jobs:
+  components-security-scan:
+    executor: security/node
+    steps:
+      - checkout
+      - security/detect_secrets
+      - security/scan_dependencies:
+          pkg_manager: npm
+  examples-security-scan:
+    executor: security/node
+    steps:
+      - checkout
+      - security/scan_dockerfile:
+          dockerfile_dir: ./examples
+
+workflows:
+  scan-and-test:
+    jobs:
+      - components-security-scan:
+          filters: *pr-filters
+      - examples-security-scan:
+          filters: *pr-filters
```

Comment:
Why not perform these on every branch? 👀

Reply:
This is now overhauled, please take a look.